CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,81 @@
+# Up-For-Grabs Code of Conduct
+
+This document outlines how contributors to the Up-For-Grabs project
+should interact with each other, and the process for resolving issues.
+
+Projects listed on Up-For-Grabs are not covered by this code of conduct.
+Inquiries about a specific project should be addressed according to that
+project's governance rules.
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
```

Comment:
Is it very welcoming to single out people for the non-immutable characteristics?
Is there no universality to it, or do you for some reason need to do it for just these ones, is that the kind of community this is?